### DISCUSSION TITLE
4.7.1

### DISCUSSION THREAD

**Discussion by ywmoyue** [ywmoyue/biliuwp-lite#750] [Announcements]
⚠ x64_native的版本当前无法编译，如有需要请等待后续临时版本
⚠ 由于存储数据方式的变更，上一版本新增的内容过滤设置项将清空，如需要保留请更新前使用导出设置功能备份并在更新后重新导入

----

#### ADDED
- 播放器设置中播放模式增加播完暂停选项 #736 
- 支持按键快捷键自定义，支持区分设置按键点击行为和按住行为 #730 #728
- 添加 关闭推荐页顶部横版推广条 的选项 #747 
#### CHANGED
- 增加快速收藏设置项，默认启用快速收藏 #722
- 引入SQLite数据库保存uwp无法保存的较大的数据 #730
    - ⚠ 上一版本新增的内容过滤设置项将清空
#### FIXED
- 修复内容过滤设置规则项在Win10中样式错误 #716
- 修复带播放列表的视频未打开过播放列表时播放结束闪退 #720  #740 
- 修复在部分版本系统中导出设置功能无效 #718
- 修复动态页专栏动态tab和追番追剧tab动态无法点击 #737 

<hr /><em>This discussion was created from the release <a href='https://github.com/ywmoyue/biliuwp-lite/releases/tag/v4.7.1'>4.7.1</a>.</em>

**Comment by KennethWuWu** [comment]
大佬，后续还会有x86版本的吗？

**Reply by ywmoyue** [reply]
编译一个x86出来是可以的，有什么使用场景需要x86而不是x64吗

**Reply by KennethWuWu** [reply]
我有一个x86的win板🥳

**Reply by ywmoyue** [reply]
https://github.com/ywmoyue/biliuwp-lite/releases/tag/v4.7.1.202
